src/protobuf-net/Meta/MetaType.cs

```diff
@@ -2080,11 +2084,13 @@ public bool IsGroup
                     if (parsed.HasValue)
                     {
                         if (parsed.Value == 0) continue;
-                        NewLine(builder, indent + 1).Append(member.Name).Append(" = ").Append(parsed.Value).Append(';');
+                        NewLine(builder, indent + 1).Append(isEnumNamePrefixSupported ? $"{enumNamePrefix}{member.Name}" : member.Name).Append(" = ").Append(parsed.Value).Append(';');

```

Comment:
there's a semantic thing here for me to think about - "what should happen if the name was already explicitly specified? is it prefixed? (probably not)" - not 100% sure we know that currently; a job for me to worry about